Universal/Tests/CodeAnalysis/MixedBooleanOperatorUnitTest.inc

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
The examples are currently grouped to have matching “not ok” and “ok” examples right beneath each other (i.e. first the one without parentheses and then the two possible variants with parentheses).

I can reorder the tests if you insist, but I personally find that harder to understand. It would also shift all line number if tests are added after the fact.

Reply:
Okay, let's leave them in the order they are then, but in that case, what about having trailing `// Okay.` / `// Error.` comments ? That way it is still clearer what is being tested in each line.

Reply:
> It would also shift all line number if tests are added after the fact.

It is only for the initial set of tests. Tests added at a later point go after with a comment referencing the issues they relate to.